### PR TITLE
[CSFS-37] Database Design For A transportation Company

### PR DESCRIPTION
**Pull request by Tocyn**
Screenshot 2021-01-30 at 22 58 36
Screenshot 2021-01-30 at 22 58 58
Screenshot 2021-01-30 at 22 58 27

### CODE REVIEW

File: submissions/oluwatosinOke/database/transportation.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Point_of_embarkation VARCHAR (50) NOT NULL,
+PRIMARY KEY (trip_id),
+FOREIGN KEY (passenger_id) REFERENCES passengers(passenger_id)
+);
+
+-- create table accidents
+CREATE TABLE accidents (
+id INT NOT NULL AUTO_INCREMENT,
+passenger_id INT NOT NULL,
+survived VARCHAR (50) NOT NULL,
```
Comment (adaugochi) [inline submissions/oluwatosinOke/database/transportation.sql:31]: `survived` should be either `TINYINT` or `BOOLEAN` not `VARCHAR`

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Tocyn** [comment]
@adaugochi 
I have changed the survived column data type.
Kindly Review.

**Review by adaugochi — CHANGES_REQUESTED** [review]
I hope you rerun this query when you made your changes. If no, kindly drop the database and rerun this query again. If yes, let me know so I approve

**Comment by Tocyn** [comment]
@adaugochi 
i did rerun the query.
kindly review

**Comment by Tocyn** [comment]
@adaugochi 
i did rerun the query.
kindly review

**Review by adaugochi — APPROVED** [review]

**Pull request merged**